templates/csharp/api.mustache

```diff
@@ -60,5 +55,33 @@ namespace {{packageName}}.{{apiPackage}}
         {{/supportsAsync}}
         {{/operation}}
     }
+    
     {{/operations}}
-}
+    {{#operations}}
```

Comment:
I'd love to have the interface in a separate class. The implementation might have other namespaces that the interface doesn't need to know about.

However, if you want to go forward with 1 single class, I'd put the interface on top and the implementation on the bottom 👍

Reply:
Looking at the documentation it takes a substantial effort to have the generator create new files rather than modifying templates for existing ones. @jillingk maybe we can make a ticket to do this somewhere in the future as a nice-to-have?